cli/flox-rust-sdk/src/providers/services.rs

```diff
@@ -79,6 +112,44 @@ impl From<ManifestServices> for ProcessComposeConfig {
     }
 }
 
+impl Serialize for ProcessComposeConfig {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        let mut processes = self.processes.clone();
+        // Inject an extra process to prevent `process-compose` from exiting when all services are stopped.
+        processes.insert(
+            PROCESS_NEVER_EXIT_NAME.to_string(),
+            generate_never_exit_process(),
+        );
```

Comment:
I think this is a detail that is worth documenting, at least internally, eg. as part of the module documentation.

Reply:
Done